llvm-external-projects/iree-dialects/test/Dialect/iree_linalg_ext/tiling.mlir

```diff
@@ -15,8 +14,13 @@ func.func @scatter_tiling(
     } -> tensor<?x?xf32>
   return %0 : tensor<?x?xf32>
 }
-//   CHECK-DAG: #[[MAP0:.+]] = affine_map<(d0)[s0, s1] -> (10, -d0 + s1)>
-//   CHECK-DAG: #[[MAP1:.+]] = affine_map<(d0)[s0, s1] -> (20, -d0 + s1)>
+transform.sequence failures(propagate) {
+^bb1(%module_op: !transform.any_op):
+  %0 = transform.structured.match ops{["iree_linalg_ext.scatter"]} in %module_op : (!transform.any_op) -> !transform.any_op
+  %1, %loops:2 = transform.structured.tile_using_for %0 [10, 20] : (!transform.any_op) -> (!transform.any_op, !transform.any_op, !transform.any_op)
```

Comment:
I verified that it is using `scf::tileUsingSCFForOp` method.

https://github.com/llvm/llvm-project/blob/e775ba384efe31928bac796b2a4d388a1c298c5e/mlir/lib/Dialect/Linalg/TransformOps/LinalgTransformOps.cpp#L2660-L2661